matchingengine/OrderBook_test.go

```diff
@@ -139,41 +594,222 @@ func TestLoadOldBid(t *testing.T) {
 		stockId:     stockID,
 		askChan:     make(chan *models.Ask),
 		bidChan:     make(chan *models.Bid),
-		asks:        mockAskQueue,
-		bids:        mockBidQueue,
-		askStoploss: mockAskStoplossQueue,
-		bidStoploss: mockBidStoplossQueue,
-		depth:       mockDepth,
+		asks:        testAskQueue,
+		bids:        testBidQueue,
+		askStoploss: testAskStoplossQueue,
+		bidStoploss: testBidStoplossQueue,
+		depth:       testDepth,
 	}
 
-	limitBid := makeBid(1, stockID, models.Limit, stockQuantity, stockPrice, "")
+	ob.LoadOldAsk(ask)
+	ob.LoadOldBid(bid)
+	models.LoadStocks()
+	time.Sleep(time.Second * 2)
```

Comment:
wg.SyncGroup like mentioned above

Reply:
time.Sleep was not necessary my bad. so just removing it works.